docs/src/main/asciidoc/extension-codestart.adoc

```diff
@@ -174,6 +174,63 @@ NOTE: `codestart.yml` is the only required file.
 * `./base` contains all the files that will be processed independently of the specified language
 * `./[java/kotlin/scala]` contains all the files that will be processed only if the specified language has been selected (overriding base)
 
+=== Dynamic Config Keys in Codestart
+
+[source,bash]
+----
+gen-info.time: time of generation (in millis)
+
+input.selected-extensions[].name|description|guide: list of selected extensions with info
+
+input.selected-extensions-ga: Set of String with the list of extensions groupId:artifactId, usefull for dynamic codestarts depending on selected extensions
+
+input.provided-code[].name|tags|title|description|related-guide: list of selected codestarts with info
+
+----
+
+=== Static Config Keys in Codestart
+
```

Comment:
Instead of having the legacy enum like `BOM_GROUP_ID`, just add the key and description (like above)